elimar/modules/users/views.py

```diff
@@ -23,3 +23,18 @@ class UsersList(ListView):
     model = User
     paginate_by = 20
     template_name = "users/users_list.html"
+
+
+class UserDetailsEdit(UpdateView):
+    model = User
+    fields = [
```

Comment:
Shouldn't we have a form or something?

Reply:
I guess based on model, We don't needed it https://docs.djangoproject.com/en/4.1/ref/class-based-views/generic-editing/#django.views.generic.edit.UpdateView 🤔